src/main/java/spoon/support/compiler/jdt/JDTCommentBuilder.java

```diff
@@ -558,7 +558,7 @@ public void scan(CtElement element) {
 
 		if (!spoonUnit.getDeclaredTypes().isEmpty()) {
 			findCommentParentScanner.scan(spoonUnit.getDeclaredTypes());
-		} else if (spoonUnit.getDeclaredModule() != null) {
+		} else if (spoonUnit.getDeclaredModuleReference() != null) {
 			findCommentParentScanner.scan(spoonUnit.getDeclaredModule());
```

Comment:
shouldn't it be changed to `spoonUnit.getDeclaredModuleReference().getDeclaration()`?

Reply:
So you could actually keep `getDeclaratedModule()` both in the test `!= null` and after. I'd prefer a consistent solution: `declaredModule` or `declaredModuleReference` but not both.